README.md

```diff
@@ -7,6 +7,12 @@ The components in Cardboard are also used for building the experiences in [Azure
 
 The 3D visualization components in this library leverage the fantastic [BabylonJS](https://www.babylonjs.com/) library under the hood.  If you haven't used it yet, we can't say enough great things about the library, definitely check it out!
 
+Viewer Mode
```

Comment:
IMO we should bump these to headings & group them under a '3D Scenes Studio' heading.  They feel like they're floating out of context to me.

![image](https://user-images.githubusercontent.com/18181049/173636692-8fedd71c-7ef5-4e58-a35b-ea10ced2f500.png)